### PR TITLE
[chore]: v16 updated chanelog

### PR DESCRIPTION
**Pull request by stackman27**
Closes: #XXX

## What is the purpose of the change
updated changelog for v16 upgrade

### CODE REVIEW

**Comment by stackman27** [comment]
Please lmk if i missed anything. Happy to add it :))

**Review by mattverse — APPROVED** [review]

File: CHANGELOG.md
Comment (AlpinYukseloglu) [inline CHANGELOG.md:45]:
```suggestion
- Concentrated Liquidity module (x/concentrated-liquidity). Concentrated liquidity is a gamechanging new pool model that will bring unprecedented capital efficiency to Osmosis. The initial version of our design is heavily influenced by Uniswap V3, although our implementation has a number of important differences.
```

File: CHANGELOG.md
Comment (AlpinYukseloglu) [inline CHANGELOG.md:46]:
```suggestion
```

File: CHANGELOG.md
Comment (AlpinYukseloglu) [inline CHANGELOG.md:47]:
```suggestion
- Implement Cosmwasmpool module (x/cosmwasmpool). The CosmWasm Pool Module is an extension for the Osmosis pools, aiming to create a custom module that allows users to create and manage liquidity pools backed by CosmWasm smart contracts. 
```

File: CHANGELOG.md
Comment (AlpinYukseloglu) [inline CHANGELOG.md:88]:
```suggestion
  * [#4629](https://github.com/osmosis-labs/osmosis/pull/4629) Add amino proto annotations
```

File: CHANGELOG.md
Comment (AlpinYukseloglu) [inline CHANGELOG.md:101]: Is this actually an API break, or should it be under misc improvements instead?

**Review by AlpinYukseloglu — APPROVED** [review]
Looks good but left a few comments (most important one is wording on CL release). Approving to not block merge but would like at least an ack on comments!

File: CHANGELOG.md
Comment (p0mvn) [inline CHANGELOG.md:52]: Could you please add that validators will have to update the wasm cache during the upgrade due to this:

https://osmosis-network.slack.com/archives/C027ELA290B/p1685870833358839?thread_ts=1685716114.561639&cid=C027ELA290B

File: CHANGELOG.md
Comment (p0mvn) [inline CHANGELOG.md:54]: Isn't cosmawsm cherry the same as "bump wasmvm to v1.2 and add cosmwasm_1_2 capability"?

File: CHANGELOG.md
Comment (p0mvn) [inline CHANGELOG.md:43]: Could you please also review our v16 upgrade handler and make sure that we specify all parameter updates in v16 here?

Comment (p0mvn) [reply CHANGELOG.md:43]: Please ping me on Slack once complete

Comment (stackman27) [reply CHANGELOG.md:43]: updated!

File: CHANGELOG.md
Comment (p0mvn) [inline CHANGELOG.md:98]: nit: I think indentation broke here

Comment (stackman27) [reply CHANGELOG.md]: oops fixed

File: CHANGELOG.md
Comment (p0mvn) [inline CHANGELOG.md:118]: This is duplicated under "Dependencies"

Comment (p0mvn) [reply CHANGELOG.md:118]: I think we should remove this

File: CHANGELOG.md
Comment (p0mvn) [inline CHANGELOG.md:120]: Also duplicated
```suggestion
```

Comment (p0mvn) [inline CHANGELOG.md:120]: this as well
```suggestion
```

File: CHANGELOG.md
Comment (stackman27) [inline CHANGELOG.md:118]:
```suggestion
```

File: CHANGELOG.md
Comment (p0mvn) [inline CHANGELOG.md:130]: duplicated
```suggestion
```

File: CHANGELOG.md
Comment (p0mvn) [inline CHANGELOG.md:86]: I don't see this change present

**Review by p0mvn — APPROVED** [review]
Nice work. LGTM once 2 final comments are addressed

File: CHANGELOG.md
Comment (stackman27) [inline CHANGELOG.md:132]:
```suggestion
  * [#5000](https://github.com/osmosis-labs/osmosis/pull/5000) osmomath.Power panics for base < 1 to temporarily restrict broken logic for such base.
  * [#4336](https://github.com/osmosis-labs/osmosis/pull/4336) Move epochs module into its own go.mod
```

File: CHANGELOG.md
Comment (stackman27) [inline CHANGELOG.md:133]:
```suggestion
```

File: CHANGELOG.md
Comment (stackman27) [reply CHANGELOG.md:86]: we move it here: https://github.com/osmosis-labs/osmosis/pull/5418/files#diff-06572a96a58dc510037d5efa622f9bec8519bc1beab13c9f251e97e657a9d4edR98

**Pull request merged**
